### PR TITLE
Don't return draft bids in the /cyclepositions/bidders response

### PR DESCRIPTION
**Pull request by mjoyce91**
Matches FSBid functionality of not returning draft bids in the response

### CODE REVIEW

File: src/services/bids.js [hunk 1]
```diff
@@ -33,10 +33,11 @@ async function get_bid(cp_id, perdet_seq_num) {
   }
 }
 
-async function get_bids_by_cp(query) {
+async function get_bids_by_cp(query, excludeDraft = false) {
   const cp_id = _.get(query, 'request_params.cp_id');
   console.log(`Trying to get bid for cp_id=${cp_id}`)
-  const bids = await Bids
+  let bids = await Bids
+    .where(...(excludeDraft ? ['bs_cd', '<>', 'W'] : ['cp_id', cp_id])) // redundant, but best way I could conditionally insert this where statement
```
Comment (elizabeth-jimenez) [inline src/services/bids.js:40]: So i may learn: What does these `<>` beautiful symbol mean?

Comment (mjoyce91) [reply src/services/bids.js:40]: "does not equal"

Comment (elizabeth-jimenez) [reply src/services/bids.js:40]: 😍  thanks!

File: src/services/bids.js [hunk 2]
```diff
@@ -33,10 +33,11 @@ async function get_bid(cp_id, perdet_seq_num) {
...
     .where('cp_id', cp_id)
```
Comment (elizabeth-jimenez) [inline src/services/bids.js:41]: Are we able to just remove line 41 now? Seems to work

**Review by elizabeth-jimenez — APPROVED** [review]
v.cool.job

File: src/services/bids.js (hunk 2 above)
Comment (mjoyce91) [reply src/services/bids.js:41]: 41 still has to stay. Forcing the falsey value of the ternary was just the only way I could conditionally add this `where` statement

Comment (elizabeth-jimenez) [reply src/services/bids.js:41]: rippp. I was testing with a single cp_id; thanks Mike!

**Pull request merged**